emission/net/api/cfc_webapp.py

```diff
@@ -514,15 +519,13 @@ def resolve_auth(auth_method):
     except:
         webserver_log_config = json.load(open("conf/log/webserver.conf.sample", "r"))
 
+    logging.config.dictConfig(webserver_log_config)
     logging.debug("attempting to resolve auth_method")
     auth_method = resolve_auth(auth_method)
 
     logging.debug(f"Using auth method {auth_method}")
     print(f"Using auth method {auth_method}")
 
-    logging.config.dictConfig(webserver_log_config)
-    logging.debug("This should go to the log file")
```

Comment:
Is there a reason you removed this line?

Reply:
Didn't really think about it too much. Just seemed unneeded, `logging.debug("attempting to resolve auth_method")` follows the configuration, so that showing up would indicate it instead.